### ISSUE TITLE
Radio keyboard does not work as expected 

### ISSUE THREAD

**Issue by jrr997** [ant-design/ant-design#40703]
### Reproduction link

[https://ant.design/components/radio-cn](https://ant.design/components/radio-cn)

### Steps to reproduce

1.打开reproduction链接，找到示例“Radio.Group 组合 - 配置方式”。

2.示例有4个Radio.Group，点第一个Radio.Group随意选项，使其获得焦点(此时点击键盘方向键可以切换Option)

3.点击Tab切换到下一Radio.Group。

### What is expected?

焦点应该在下一个Radio.Group中。

### What is actually happening?

下一个Radio.Group没有获得焦点。

| Environment | Info |
| --- | --- |
| antd | 5.2.0 |
| React | 18 |
| System | win10 |
| Browser | chrome |

---
4.x也有这个问题

**Comment by MadCcc** [comment]
https://codesandbox.io/s/adoring-wilbur-x46tzt?file=/src/App.tsx
原生的也这样，input `type="radio"` 无法通过 tab 切换

**Comment by jrr997** [comment]
> https://codesandbox.io/s/adoring-wilbur-x46tzt?file=/src/App.tsx 原生的也这样，input `type="radio"` 无法通过 tab 切换

radio.group可以通过tab切换
https://codesandbox.io/s/brave-bouman-7pr9vl

**Comment by aojunhao123** [comment]
@afc163 大佬，这个issue可以关了，在[#52076]里已经修复这个问题
